cdap-ui/app/cdap/components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/LiveJSONButton/index.tsx

```diff
@@ -14,35 +14,55 @@
  * the License.
  */
 
-import Button from '@material-ui/core/Button';
+import * as React from 'react';
+
 import withStyles, { StyleRules } from '@material-ui/core/styles/withStyles';
-import Tooltip from '@material-ui/core/Tooltip';
+
 import CodeIcon from '@material-ui/icons/Code';
-import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import { LiveViewMode } from 'components/PluginJSONCreator/Create/PluginJSONMenu';
+import Tooltip from '@material-ui/core/Tooltip';
+import classnames from 'classnames';
 
 const styles = (theme): StyleRules => {
   return {
     buttonTooltip: {
-      fontSize: '14px',
-      backgroundColor: theme.palette.grey[500],
+      fontSize: '13px',
+      backgroundColor: theme.palette.grey[50],
+    },
+    liveJSONButton: {
+      '&:focus': {
+        outline: 'none',
+      },
+    },
+    clickedButton: {
+      backgroundColor: 'rgb(26,115,232,0.2)',
```

Comment:
See previous comment about getting this from theme.palette

Reply:
done